webapp/src/Controller/Jury/ClarificationController.php

```diff
@@ -260,12 +253,13 @@ protected function getProblemShortName(int $probid, int $cid) : string
     protected function getClarificationFormData(Team $team = null) : array
     {
         $em = $this->getDoctrine()->getManager();
+        $sp = function($lteam) {sprintf("%s (t%s)", $lteam->getEffectiveName(), $lteam->getTeamid());};
```

Comment:
This probably should be done differently

Reply:
Maybe put it in a separate function? Otherwise you need the `return` at least.

PHP 7.4+ could do:

```php
$sp = fn($lteam) => sprintf("%s (t%s)", $lteam->getEffectiveName(), $lteam->getTeamid());
```

Reply:
Ok... I'll take it just out of this PR